### DISCUSSION TITLE
Open Freeze Center v3.2.1

### DISCUSSION THREAD

**Discussion by YoCodingMonster** [YoCodingMonster/OpenFreezeCenter#144] [General]
In this Pre-release

Cleaned up the code after re understanding what i did a year ago.
Fan curves work fine [EXCEPT ADVANCED], that need some work
the sudo permissions will not pop up at all, once the main file to run is started with sudo.

Opening a discussion for it where you all can put your bugs and issues.

<hr /><em>This discussion was created from the release <a href='https://github.com/YoCodingMonster/OpenFreezeCenter/releases/tag/'>Open Freeze Center v3.2.1</a>.</em>

**Comment by jdlproenca** [comment]
[MSI Prestige 16 Studio A13VF-042XES] - fan speed doesn't go down for low temperatures.

Hi! Thank you so much for this tool, I was pretty happy to finally get some control of fan speed in Ubuntu for my laptop. However, I'm struggling to be able to get 0% CPU fan speed for low temperatures:
-  With Advanced, it seems like the option at 60ºC is the one that is always active for temperatures below 69ºC, even at 50ºC, then at 70ºC CPU temperature it triggers correctly the 70ºC fan option. Sadly, the laptop gets relatively hot at 6XºC temperatures, so setting 0% for 60ºC is not a good workaround for me.
- I know you mention that Advanced still needs some work, but otherwise I'm not able to get 0% fan, the Slowest option is at 10%, I guess because the baseline I get with Auto or Normal is 40%. 

![image](https://user-images.githubusercontent.com/34271399/235963648-96bac80a-7aed-4372-ab98-614ccb74b021.png)
![image](https://user-images.githubusercontent.com/34271399/235963799-0005832d-7aca-41de-a234-6fab48c1c66d.png)

Any clues? Thank you

**Comment by O3Dv** [comment]
When I go into the console and run the command to install, and nothing happens, no error messages, just nothing, a file is created, but i cant tell which one.

**Reply by O3Dv** [reply]
Using arch linux and an msi stealth 15m
